com.unity.sg2/Documentation~/Hexagon-Grid-Node.md

```diff
@@ -0,0 +1,23 @@
+# Hexagon Grid Node
+
+## Description
+Generates a hexagonal grid based on input **UV**. The scale of the grid is controlled by input **Scale**. In addition to grid tile outlines, the nodes also outputs the distance from edges, and a unique value or ID for each tile.
```

Comment:
Scale is a multiplier for the input UVs. The input UVs are a property of the mesh that the shader is applied to, so no, there aren't any units because the UVs vary from mesh to mesh. I can go into more detail about this concept with you if you'd like.